packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -1159,6 +1163,37 @@ class Viewport implements IViewport {
         viewPlaneNormal,
         focalPoint
       );
+      if (actorEntry?.vtkPlanes && actorEntry?.outline) {
```

Comment:
This is getting called on EVERY reset camera while it needs to only run for when it is needed (scroll), so in the setCamera in the viewport.ts you should have some logic similar to following that identify if it is an in-plane (pan, zoom) or out of plane and only when out of plane it modifies the planes

```
 if (oldFocalPoint && focalPoint) {
        const currentViewPlaneNormal = <Point3>vtkCamera.getViewPlaneNormal();
        const deltaCamera = <Point3>[
          focalPoint[0] - oldFocalPoint[0],
          focalPoint[1] - oldFocalPoint[1],
          focalPoint[2] - oldFocalPoint[2],
        ];

        const cameraModifiedInPlane =
          Math.abs(vtkMath.dot(deltaCamera, currentViewPlaneNormal)) < 1e-2;

        if (!cameraModifiedInPlane) {
          this.updateClippingPlanesForActors(updatedCamera);
          this.hasPerformedInitialClipping = true;
        }
      }
```